malaria24/ona/models.py

```diff
@@ -63,11 +63,17 @@ def alert_new_case(sender, instance, created, **kwargs):
             instance.facility_code,))
 
     for ehp in ehps:
-        if ehp.phone_number:
+        if ehp.phone_number and ehp.email_address:
```

Comment:
Considering the EHP model wasn't added in this PR but in #4 which already landed, I'm going to do that in a separate PR.